packages/cluster-ui/src/transactionDetails/transactionDetails.tsx

```diff
@@ -80,6 +82,16 @@ export class TransactionDetails extends React.Component<
     this.setState({ pagination: { ...pagination, current } });
   };
 
+  calculateTotalWorkload = (statements: Statement[]) => {
```

Comment:
Is this the same function that was in `statementsPage.tsx`? If you extract it to a utility it might be more compelling to memoize it and probably write a test for it.

Reply:
the function itself does the same thing, but statements on this file is `Statement[]` and on the file `statementsPage.tsx` is a `AggregateStatistics[]`